src/model/user-settings/UserSettings.ts

```diff
@@ -477,30 +523,20 @@ export class UserSettings implements UserSettings {
 
     private async updateFontButtons(): Promise<void> {
         for (let index = 0; index < UserSettings.fontFamilyValues.length; index++) {
-            this.fontButtons[index].className = this.fontButtons[0].className.replace(" active", "")
+            this.fontButtons[index].className = this.fontButtons[index].className.replace(" active", "")
         }
         if (this.userProperties) {
             if (this.fontButtons[await this.userProperties.getByRef(ReadiumCSS.FONT_FAMILY_REF).value]) this.fontButtons[await this.userProperties.getByRef(ReadiumCSS.FONT_FAMILY_REF).value].className += " active"
         }
     }
 
-    private updateViewButtons(): void {
-        if (this.viewButtons) {
```

Comment:
Some of the control element checks need to be re-added. 

If you call `D2Reader.scroll(true)` in the API test page an error occurs since no `viewButtons` exist.

Reply:
thank you, i missed that